WalletWasabi.Fluent/ViewModels/Wallets/Settings/WalletCoinJoinSettingsViewModel.cs

```diff
@@ -113,6 +122,25 @@ public WalletCoinJoinSettingsViewModel(UiContext uiContext, IWalletModel walletM
 					}
 				});
 
+		this.WhenAnyValue(x => x.ExcludeCoinFromCoinjoinThreshold)
+			.Skip(1)
+			.Throttle(TimeSpan.FromMilliseconds(2000))
```

Comment:
2000 ms here instead of 1000 because I want to let time to the user to write the full number. It is not important to be instant anyway because the user has no way of verifying this info right away